content/source/docs/enterprise/users-teams-organizations/entitlements.html.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: enterprise2
+page_title: "entitlements - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-users-teams-organizations-entitlements"
+---
+
+# Entitlements
```

Comment:
I'm not convinced that it makes sense for this page to exist, since there's no non-API way for orgs to "see" their entitlements. If I were going to write a page like this, I'd probably make it more "friendly" and describe a brief list of the entitlements and link to the API and the marketing plans page.